src/clj/puppetlabs/http/client/async.clj

```diff
@@ -4,137 +4,46 @@
 ;; In the options to any request method, an existing SSLContext object can be
 ;; supplied under :ssl-context. If this is present it will be used. If it's
 ;; not, the wrapper will attempt to use a set of PEM files stored in
-;; (:ssl-cert :ssl-key :ssl-ca-cert) to create the SSLContext. It is also
```

Comment:
Huh, weird.  Not sure why that was still in there.  Doesn't look like we had support for this still so good catch.  :smiley_cat: